specify/specify7
# Add formMeta menu in "View in Forms" and add unload protect to "Use Localized Field Labels"

Fixes https://github.com/specify/specify7/issues/2514
Expansion of https://github.com/specify/specify7/pull/3594

Adds a formMeta button the "View in Forms" in query results (https://github.com/specify/specify7/pull/3594) and adds an unload protect so toggling Localized Fields in Form Meta doesn't erased unsaved queries



### Checklist

      and self-explanatory (or properly documented)

### Testing instructions




- Create a new query
- Run the query and click "Browse in Forms"
- [ ] Check that the form meta button is there
- [ ] Make sure the settings within form meta work as expected
- Check Unload Protects:
  - Click Form Meta -> Form Definition
  - [ ] Click close and make sure no Unload Protect appears
  - Click Form Meta -> Form Definition and toggle "Use Localized Field Labels"
  - [ ] Click close and make sure an Unload Protect appears
  - Save the query and try again
  - [ ] Click close and make sure no Unload Protect appears
  - [ ] Do the same for the Form Meta button in Data Entry
